pkg/agents/team3/emotions.go

```diff
@@ -8,11 +8,55 @@ import (
 // Function decides if we read a message or we don't, depends on our level of stubbornness, return is bool
 // Stubborness of 20 means probability of 0.2 that we don't read message
 
-//func read(a *CustomAgent3) bool {
-//	rand.Seed(time.Now().UnixNano()) //creates seed to rand
-//	var random = rand.Intn(100)
+func read(a *CustomAgent3) bool {
```

Comment:
Prefer this:
```
func (a *CustomAgent3) read() bool {
```
This makes it a member function of the struct so you don't have to pass the agent.